source/iopipe/json/serialize.d

```diff
@@ -500,23 +500,30 @@ unittest {
 }
 
 /**
- * Expect the given JSONItem to be a specific token.
+ * Expect the given Element or token to be a specific token.
  * Parameters:
- *      item: The item to check
+ *      item: The item/token to check
```

Comment:
The parameter in the token function is named (or should be named) `token`

Reply:
That's how you do it, thanks. It will show up with 2 prototypes over a common description, e.g. https://dlang.org/phobos/std_algorithm_searching.html#.find.2